docker/conda_build.dockerfile

```
@@ -1,92 +1,74 @@
-ARG ubuntu_version=22.04
-
+ARG ubuntu_version=24.04
 FROM ubuntu:${ubuntu_version} AS pyne-deps
 
-# Ubuntu Setup
-ENV TZ=America/Chicago
-RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+# Set environment variables
+ENV TZ=America/Chicago \
+    HOME=/root \
+    PATH=/opt/conda/bin:$PATH \
+    LD_LIBRARY_PATH=/opt/conda/lib:$LD_LIBRARY_PATH
```

I think docker now complains (warning) that `LD_LIBRARY_PATH` is not yet defined, so it's probably better to do this:
```suggestion
    LD_LIBRARY_PATH=/opt/conda/lib
```